src/fragments/lib/push-notifications/android/setup_push_service/setup-fcm.mdx

```diff
@@ -14,7 +14,7 @@ Next, you will need to access your **ServerKey** (Referred to as **ApiKey** in t
 - Select the **Cloud Messaging** tab.
 - Select the three vertical dots next to **Cloud Messaging API (Legacy)**, then select **Manage API in Google Cloud Console**. ![Firebase Console](/images/push-notifications/firebaseconsole.png)
 - In the new tab, select the **Enable** button.
-- Return to the previous tab and refresh the page.
-- Copy the **Token** next to **Server key**
+- Return to the **Cloud Messaging** tab and refresh the page.
```

Comment:
I believe it was "return to the previous tab" because clicking "manage API in google cloud console" opens a new browser tab.  In fact the new tab is also a cloud messaging tab.

Reply:
I think the "previous page" is more meaningful than tab.

Reply:
I went with "previous page" and I added a screenshot that really drives it home 😄